modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/BlogFeature.cs

```diff
@@ -0,0 +1,28 @@
+﻿using JetBrains.Annotations;
+using System;
+using Volo.Abp;
+using Volo.Abp.Domain.Entities.Auditing;
+
+namespace Volo.CmsKit.Blogs
+{
+    public class BlogFeature : FullAuditedAggregateRoot<Guid>, IEquatable<BlogFeature>
+    {
+        public BlogFeature(Guid blogId, [NotNull] string featureName, bool enabled = true)
+        {
+            BlogId = blogId;
+            FeatureName = Check.NotNullOrWhiteSpace(featureName, nameof(featureName));
+            Enabled = enabled;
+        }
+
+        public Guid BlogId { get; protected set; }
+
+        public string FeatureName { get; protected set; }
+
+        public bool Enabled { get; set; } = true;
+
+        public bool Equals(BlogFeature other)
+        {
+            return BlogId == other?.BlogId && FeatureName == other?.FeatureName;
+        }
```

Comment:
I could not see the usage of this, can be removed.

Reply:
This is required to compare while using `.AddIfnotExist()` extension. We need **BlogId** and **FeatureName** combination comparison instead of reference comparison.